src/components/dashboard/dashboard-view.js

```diff
@@ -25,4 +27,16 @@ class DashboardView extends Component {
   }
 }
 
-export default translate(['translations'], translateOptions)(DashboardView);
\ No newline at end of file
+function mapStateToProps(state) {
```

Comment:
remove unwanted code blocks.. These can be added when needed

Reply:
Cool.